css-box-model/styles.css

```diff
@@ -0,0 +1,11 @@
+button.language {
+  font-size: 24px;
+  background-color: lightgray;
+  margin-left: 12px;
+  margin-right: 12px;
+  padding-left: 16px;
+  padding-right: 16px;
+  padding-top: 6px;
```

Comment:
Where's the bottom padding?

---

![image](https://user-images.githubusercontent.com/16143771/167921877-0d29391d-3fdb-4ede-bf4d-cf5631feb7ca.png)